src/main/kotlin/com/example/plugins/PoolInfoController.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.plugins
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import okhttp3.Credentials
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+import java.io.IOException
+
+class PoolInfoController {
+
+    private data class JsonData (val t1: Float, val t2:Float, val t3: Float, val p1: Float)
```

Comment:
Please change the name of this class to something more self-descriptive (i.e. _PoolInfoData_)